upgrade/Upgrade_Only_CSM_with_iuf.md

```diff
@@ -34,10 +59,14 @@ using IUF.
 
 1. Management rollout
 
+   > **NOTE** The CSM services upgrade and validate CSM health are automatically executed in a hook run before the first management node is rolled out.
+
    Follow the IUF [Management rollout](../operations/iuf/workflows/management_rollout.md) instructions.
 
 1. Deploy product
 
+   > **NOTE**  Networking and CoreDNS anti-affinity are applied and the Kubernetes control plane is upgraded in a hook automatically executed at the end of `deploy-product`.
```

Comment:
Same comment on wording as in the other location.

Reply:
Fixed wording.